build.gradle

```diff
@@ -37,9 +37,9 @@ dependencies {
   compile gradleApi()
   compile localGroovy()
 
-  compile 'com.squareup.spoon:spoon-runner:1.1.2'
+  compile 'com.squareup.spoon:spoon-runner:1.1.5'
 
-  compile 'com.android.tools.build:gradle:0.14.1'
+  compile 'com.android.tools.build:gradle:1.0.0'
```

Comment:
Can't remember the error but I had a problem in my project when I went up to 1.1